db/migrate/20220707150906_create_cloud_database_server.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCloudDatabaseServer < ActiveRecord::Migration[6.0]
+  def change
+    create_table :cloud_database_servers do |t|
+      t.string :name
+      t.string :ems_ref
+      t.bigint :ems_id
+      t.string :server_type
+      t.string :region
```

Comment:
@agrare Should this be region, or provider_region?  I'm wondering if we even need this if we have the EMS.

Reply:
Yeah from the example data region is `East US` which is already selected when you create the provider